code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -97,23 +101,15 @@
 	user.visible_message("<span class='suicide'>[user] begins to choke on \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return OXYLOSS//ironic. he could save others from oxyloss, but not himself.
 
-/obj/item/reagent_containers/hypospray/medipen/attack(mob/M, mob/user)
-	if(!reagents.total_volume)
-		to_chat(user, "<span class='warning'>[src] is empty!</span>")
-		return
-	..()
-	if(!iscyborg(user))
+/obj/item/reagent_containers/hypospray/inject(mob/living/M, mob/user)
+	if(..())

```

Comment:
This is a kinda dirty way to do it because subtypes won't get the same return. It's a lot cleaner to do it this way.
```
	. = ..()
	if(.)

Reply:
Uhhhh, yeah that does make sense. Okay.